contrib/firejail-welcome.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+
+# This file is part of Firejail project
+# Copyright (C) 2014-2020 Firejail Authors
```

Comment:
as thils file was written in 2020, the correct copyright would probably be just 2020.